Makefile

```diff
@@ -24,7 +24,7 @@ TOOLS = \
 VERSION = $(shell git describe --tags --dirty --exact-match 2>/dev/null || git rev-parse --short HEAD)
 
 GO_LDFLAGS = \
-	-ldflags "-extldflags '-static' -X main.Version=$(VERSION)"
+	-ldflags "-X main.Version=$(VERSION)"
```

Comment:
Out of curiosity: Why did you remove this?

I saw the warnings when this was readded, but it worked for me, and this could be quite desirable, especially on Mac and Windows, where the the lzma library is less likely to exist.

Reply:
here I just removed it to make as minimal a change as possible to the way it works now, as I didn't want to cause a breaking change somewhere else. It worked in the container as well but I wasn't sure about the warnings...